<launch>
    <!-- turtle -->
    <node pkg="turtlesim" type="turtlesim_node" name="turtlesim" />

    <!-- joystick -->
    <node pkg="joy" type="joy_node" name="joystick">
        <param name="dev" type="string" value="/dev/input/js0" />
        <param name="deadzone" value="0.12" />
    </node>

    <!-- teleop -->
    <node pkg="js_turtle_teleop" type="js_turtle_teleop_node.py" name="js_turtle_teleop_node_py" output="screen">
        <param name="linear_axis_num" value="1" type="int" />
        <param name="angular_axis_num" value="3" type="int" />
        <param name="max_linear_speed" value="2.0" type="double" />
        <param name="max_angular_speed" value="2.0" type="double" />
    </node>

</launch>
